src/services/legal-services.ts

```diff
@@ -173,9 +174,9 @@ export default class LegalServices {
    * Throws an exception on error.
    */
   static async createBusiness (businessRequest: any): Promise<any> {
-    const legalApiUrl = sessionStorage.getItem('LEGAL_API_URL')
+    const legalApiUrl = useConfigurationStore().getLegalApiUrl
```

Comment:
I think you don't need this because of this:
https://github.com/bcgov/business-filings-ui/blob/927667d7cd105c27bb61eef9524ef7d26075286f/src/stores/configurationStore.ts#L133

(See the other axios calls below for examples.)

Reply:
Looks good. Resolving.